operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,404 @@
+# Editing/Deleting critical services from static ConfigMap (`rrs-mon-static`)
+
```

Comment:
Should add a table of contents here with links to the ## sections at least, to make the page easier to navigate